ACadSharp/IO/DXF/DxfStreamReader/DxfSectionReaderBase.cs

```diff
@@ -585,12 +585,6 @@ private bool readLwPolyline(CadEntityTemplate template, DxfMap map, string subcl
 						last.Bulge = this._reader.ValueAsDouble;
 					}
 					return true;
-				case 50:
-					if (last is not null)
-					{
-						last.CurveTangent = this._reader.ValueAsDouble;
-					}
-					return true;
```

Comment:
Why has this been removed?

Reply:
As before. We did not make any changes to this class.